website/docs/using-themes.md

```diff
@@ -3,20 +3,61 @@ id: using-themes
 title: Using Themes
 ---
 
-_This section is a work in progress. [Welcoming PRs](https://github.com/facebook/docusaurus/issues/1640)._
+Themes provide the consistent components for your doc sites.
+
+<!-- TODO: WIP intro -->
+
```

Comment:
i find it hard to understand themes without reading advanced guide-themes. I think we should try giving user understanding of themes first before teaching them how to use it. Not really sure whats the best approach